<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QubeHub</class>
 <widget class="QStackedWidget" name="QubeHub">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pictures/Logo.png</normaloff>pictures/Logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="currentIndex">
   <number>3</number>
  </property>
  <widget class="QWidget" name="main_view">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>670</y>
      <width>1011</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="main_exit_button">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="main_delete_button">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Delete Center</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="main_create_user_button">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Create Player</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="main_create_questionary_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Create Questionary</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="main_create_question_button">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Create Question</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>991</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pictures/question.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="create_user_view">
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1011</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="create_user_nickname_label">
       <property name="text">
        <string>Nickname</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="create_user_nickname_field"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="create_user_email_label">
       <property name="text">
        <string>E-Mail</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="create_user_email_field"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="create_user_rfid_label">
       <property name="text">
        <string>RFID Nr.:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="user_rfid_number_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;RFID Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Create Player&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>730</y>
      <width>1021</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="create_user_cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="create_user_create_button">
       <property name="text">
        <string>Create Player</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="create_question_view">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>730</y>
      <width>1021</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="create_question_cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="create_question_create_button">
       <property name="text">
        <string>Create Question</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1011</width>
      <height>488</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="create_question_questionary_label">
       <property name="text">
        <string>Questionary</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QTextEdit" name="create_question_info_text"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Create Question&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="create_question_answer_one_label">
       <property name="text">
        <string>Answer One</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QRadioButton" name="create_question_answer_three_button">
       <property name="text">
        <string>is the correct Answer</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QRadioButton" name="create_question_answer_four_button">
       <property name="text">
        <string>is the correct Answer</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="create_question_answer_four_field"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="create_question_question_label">
       <property name="text">
        <string>Question</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="create_question_answer_three_field"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="create_question_answer_two_label">
       <property name="text">
        <string>Answer Two</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="create_question_question_field"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="create_question_answer_three_label">
       <property name="text">
        <string>Answer Three</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="create_question_answer_two_field"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="create_question_answer_four_label">
       <property name="text">
        <string>Answer Four</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QRadioButton" name="create_question_answer_two_button">
       <property name="text">
        <string>is the correct Answer</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="create_question_answer_one_field"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="create_question_info_label">
       <property name="text">
        <string>Why is this Answer Correct?</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QRadioButton" name="create_question_answer_one_button">
       <property name="text">
        <string>is the correct Answer</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="create_question_questionary_box"/>
     </item>
     <item row="2" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="delete_view">
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>730</y>
      <width>1021</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="delete_cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>50</y>
      <width>1011</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <widget class="QComboBox" name="delete_questionary_box"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="delete_question_box"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="delete_question_label">
       <property name="text">
        <string>Question</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="delete_player_button">
       <property name="text">
        <string>Delete Player</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="delete_questionary_label">
       <property name="text">
        <string>Questionary</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="delete_player_box"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="delete_question_button">
       <property name="text">
        <string>Delete Question</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="delete_player_label">
       <property name="text">
        <string>Player</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="delete_questionary_button">
       <property name="text">
        <string>Delete Questionary</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Delete Center&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="create_questionary_view">
   <widget class="QWidget" name="formLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1011</width>
      <height>164</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="create_questionary_name_label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="create_questionary_category_label">
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="create_questionary_lecture_label">
       <property name="text">
        <string>Lecture</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="create_questionary_name_field"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="create_questionary_category_field"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="create_questionary_lecture_field"/>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Create Questionary&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>730</y>
      <width>1021</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="create_questionary_cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="create_questionary_create_button">
       <property name="text">
        <string>Create Questionary</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>main_exit_button</sender>
   <signal>clicked()</signal>
   <receiver>QubeHub</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
